### PR TITLE
Installation wizard version bump-up to 1.13

### PR DESCRIPTION
**Pull request by samihan-dell**
# Description
Updated Installation wizard to support CSM version 1.13

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1559 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [X] Have you added high-resolution images?

image

### CODE REVIEW

**Review by niranjan-n1 — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]
LGTM

File: content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ const CONSTANTS = {
 	CSM_HELM_V1102: "1.3.2",
```
Comment (chimanjain) [inline content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js:46]: We should remove this if we are removing the support for `CSM_HELM_V1102`.

File: content/docs/deployment/csminstallationwizard/src/static/js/tests/ui-functions.test.js [hunk 2]
```diff
@@ -72,6 +72,7 @@ const CONSTANTS = {
 	CSM_HELM_V1102: "1.3.2",
```
Comment (chimanjain) [inline content/docs/deployment/csminstallationwizard/src/static/js/tests/ui-functions.test.js:72]: We should remove this if we are removing the support for `CSM_HELM_V1102`.

File: content/docs/deployment/csminstallationwizard/src/static/js/tests/utility.test.js [hunk 3]
```diff
@@ -49,6 +49,7 @@ const CONSTANT_PARAM = {
 	CSM_HELM_V1102: "1.3.2",
```
Comment (chimanjain) [inline content/docs/deployment/csminstallationwizard/src/static/js/tests/utility.test.js:49]: We should remove this if we are removing the support for `CSM_HELM_V1102`.

File: content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js (hunk 1 above)
Comment (samihan-dell) [reply content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js]: Removed it. Thanks!

File: content/docs/deployment/csminstallationwizard/src/static/js/tests/ui-functions.test.js (hunk 2 above)
Comment (samihan-dell) [reply content/docs/deployment/csminstallationwizard/src/static/js/tests/ui-functions.test.js]: Removed it. Thanks!

File: content/docs/deployment/csminstallationwizard/src/static/js/tests/utility.test.js (hunk 3 above)
Comment (samihan-dell) [reply content/docs/deployment/csminstallationwizard/src/static/js/tests/utility.test.js]: Removed it. Thanks!

**Review by chimanjain — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**